memory-viz/package.json

```diff
@@ -9,8 +9,8 @@
         "test": "jest --no-cache",
         "test-cov": "jest --no-cache --coverage",
         "watch": "webpack --watch --config webpack.dev.js",
-        "build-dev": "tsc && webpack --config webpack.dev.js",
-        "build": "tsc && webpack --config webpack.prod.js",
+        "build-dev": "tsc && webpack --config webpack.common.js --mode development",
```

Comment:
Oh you shouldn't change this, but instead update `webpack.dev.js` and `webpack.prod.js` to account for the new configs exported from `common`. Note that we're using `webpack-merge` to update configurations, so you'll need to tweak that to preserve the behaviour for the `libraryCli` configuration.